provider/marathon/marathon_test.go

```diff
@@ -275,29 +211,21 @@ func TestMarathonLoadConfigNonAPIErrors(t *testing.T) {
 		},
 		{
 			desc: "health check labels",
-			application: marathon.Application{
-				Ports: []int{80},
-				Labels: &map[string]string{
+			application: application(
+				appPorts(80),
+				labels(map[string]string{
 					types.LabelBackendHealthcheckPath:     "/path",
 					types.LabelBackendHealthcheckInterval: "5m",
-				},
-			},
-			task: marathon.Task{
-				Host:  "127.0.0.1",
-				Ports: []int{80},
-				IPAddresses: []*marathon.IPAddress{
-					{
-						IPAddress: "127.0.0.1",
-						Protocol:  "tcp",
-					},
-				},
-			},
+				}),
+			),
+
```

Comment:
<fist-swinging>Argh, damn you unintentional blank lines!</fist-swinging>

Removed. :-)